### ISSUE TITLE
Error: GitHub Personal Access Token is not set, neither programmatically, nor using env "GH_TOKEN"

### ISSUE THREAD

**Issue by developerautomationkit** [electron-userland/electron-builder#8676]
* **Electron-Builder Version**:  

* **Node Version**: 

* **Electron Version**:
* **Electron Type (current, beta, nightly)**:

* **Target**: 

Error: GitHub Personal Access Token is not set, neither programmatically, nor using env "GH_TOKEN"

Iam trying to use a PAT

GH_TOKEN: ${{ secrets.RELEASE_PAT }} -- doesnt work ... using a fine grained  PAT with contents permission added to the repository as secret...
GH_TOKEN: ${{ secrets.GITHUB_TOKEN}} -- works ...  but I actually need the pat for pushing my release to different repo (
   
   I have this in my electron builder config yml
   
```
   directories:
  output: dist
  buildResources: buildResources

files:
  - packages/**/dist/**

win:
  target:
    - nsis

publish:
  provider: github
  owner: me
  repo: seperate_public_repo
  private: true
   
```
   
   Basically iam pushing to a private repo but want my release to be pushed in the github action to a seperate public repo

**Comment by developerautomationkit** [comment]
I just ran the command locally to push the repo directly ... which works ... but if I do my said setup in github actions it still does not...


export GH_TOKEN=my token
./node_modules/.bin/electron-builder --config electron-builder.yml --publish always

**Comment by hichemBAALI** [comment]
> I just ran the command locally to push the seperate repo directly ... which works ... but if I do my said setup in github actions it still does not... for some reason it does not recognize any kind of personal acces tokens like this ?
> 
> Does not recognize(in gh actions) : GH_TOKEN: ${{ secrets.RELEASE_PAT }}
> 
> Works (locallly): export GH_TOKEN=my token ./node_modules/.bin/electron-builder --config electron-builder.yml --publish always

Same for my case. Did you figure out any solution ?
